### ISSUE TITLE
Discussion: Implementation

### ISSUE THREAD

**Issue by jacobp100** [styled-components/css-to-react-native#3]
I’ve put a quick demo implementation using pegJS for parsing. While it doesn’t currently show the full potential, I think if we do start allowing shorthands, like `border: 1px solid`, it’ll be very useful to be using something like this. However, it could also be overkill.

How do people want to handle parsing?

**Comment by mxstbr** [comment]
Well, we use PostCSS for our styled-components code. If you find something smaller that can handle nesting etc., that'd be much better!

**Comment by jacobp100** [comment]
It still uses PostCSS to parse declarations, but the actual values for the declarations, I’ve done with pegJS (which I just changed to Nearley). I did look at using external css shorthand parsers for these things, but they don’t cover all properties, usually aren’t maintained, and don’t fit quite as nicely. I’ve got quite a lot of mileage out of doing it this way.

Would be interested to see what you think of the demo implementation.
